Dockerfile

```diff
@@ -1,17 +1,18 @@
 FROM golang:1.19-alpine3.15 as builder
```

Comment:
could bump to 3.16

Reply:
Thanks, not sure how I didn't catch that before.